### PR TITLE
#167/Fix logout from multiple tabs

### LINKED ISSUE #167: ⚛️ Fix logout from multiple tabs

### PR DESCRIPTION
**Pull request by gbudau**
How it works:

Each application instance creates a `BroadcastChannel` that subscribes to the `auth_channel`
Whenever the user logs out from one of the applications, it sends a message(the currently authenticated user object) to all the other applications subscribed to the `auth_channel`.
Whenever another application receives this message, it compares the users, and if they are the same, it logs out the user.

https://stackoverflow.com/a/58814352

close #167

### CODE REVIEW

File: webapp/src/shared/context/auth-context.tsx [hunk 1]
```diff
@@ -28,6 +28,16 @@ export const AuthProvider = ({ children }: { children: ReactNode }) => {
   }, []);
   const { data, isLoading: isDataLoading } = useData<User>(getCurrentUser);
   const navigate = useNavigate();
+  const authBroadcastChannel = useMemo(
+    () => new BroadcastChannel('auth_channel'),
+    [],
+  );
```
Comment (vdedios) [inline webapp/src/shared/context/auth-context.tsx:34]: Is this available to any webapp instance running on the same browser?

Comment (gbudau) [reply webapp/src/shared/context/auth-context.tsx:34]: Yes, each application running in a different tab or window subscribes to the `auth_channel` -> https://developer.mozilla.org/en-US/docs/Web/API/BroadcastChannel#constructor

And they can send messages to each other.

**Review by fizcamposancos — APPROVED** [review]
Simple and complex at the time.
LGTM!!

**Comment by gbudau** [comment]
Thanks!

**Pull request merged**